packages/jasmine-extensions/src/retry-failed-tests.ts

```diff
@@ -47,7 +49,19 @@ export function retryFailedTests(
 
                 if (millisecondsBetweenRetries && i != retries - 1) {
                     console.log(
-                        `Test ${spec.getFullName()} failed, attempting retry ${i + 1} in ${millisecondsBetweenRetries}ms`
+                        `Test "${spec.getFullName()}" failed, attempting retry ${i + 1} cleanup`
+                    );
+                    for (let j = 0; j < beforeAfterFns.afters.length; j += 1) {
```

Comment:
I was expecting to see "befores"->"test"->"afters" rather than "afters"->"befores"->"test". 

Does Jasmine not run the after function after a test fails so this is the "after" for the previous failed run? If so then a comment explaining that would be good. And optionally updating the console.log to use "i" instead of "i+1" or to say "from run before retry {i+1}